src/app/core/services/sqlite/sqlite.service.ts

```diff
@@ -273,6 +285,9 @@ export class SqliteService {
     if (searchCriteria.Extent?.TopLeft?.Longitude != null) {
       where.push(`lon >= ${searchCriteria.Extent.TopLeft.Longitude}`);
     }
+    if (searchCriteria.ObserverCompetence != null) {
```

Comment:
Hvordan blir dette hvis ObserverCompetence er [] ? 
Får vi da en observer_competence IN () og er det det vi vil?

Reply:
Det er ikke mulig å få en tom array med vår oppsette tror jeg. det er et array med nummere eller blir det null.